setup.py

```diff
@@ -1,26 +1,25 @@
 from setuptools import setup, find_packages
 from pyouroboros import VERSION
 
-requirements = ['docker>=3.7.0',
-                'apscheduler>=3.5.3',
-                'prometheus_client>=0.5.0',
-                'requests>=2.21.0',
-                'influxdb>=5.2.1',
-                'apprise>=0.5.2']
 
-
-def readme():
-    with open('./README.md') as f:
+def read(filename):
+    with open(filename) as f:
         return f.read()
 
 
+def get_requirements(filename="requirements.txt"):
+    """returns a list of all requirements"""
+    text = read(filename)
+    return [line for line in map(lambda l: l.split('#')[0].strip(), text.splitlines()) if line != '']
```

Comment:
All the rest looks great. This part is a bit confusing for me with the lambda as i dont see any glaring oddities in the file.
```
text = read('requirements.txt')
'docker>=3.7.0\nprometheus_client>=0.5.0\nrequests>=2.21.0\ninfluxdb>=5.2.1\napprise>=0.5.2\napscheduler>=3.5.3'
[requirement for requirement in text.split()]
['docker>=3.7.0', 'prometheus_client>=0.5.0', 'requests>=2.21.0', 'influxdb>=5.2.1', 'apprise>=0.5.2', 'apscheduler>=3.5.3']
```

Reply:
That's only there to be able to comment things out with a `#` in `requirements.txt`, but if you think that's not required, a simple `return [line.strip() for line in text.splitlines()) if line != '']` would be fine too. The comparison for `''` is there to ignore empty lines.

Reply:
cool. and just to account for all empty (None, '', False, etc) we can do filter instead. Commenting out a requirement is one thing but adding comments inline on the requirement is silly. Heres a final:
```python
def get_requirements(filename="requirements.txt"):
    """returns a list of all requirements"""
    requirements = read(filename)
    return list(filter(None, [req.strip() for req in requirements.split() if not req.startswith('#')]))
```

Reply:
Ok, the PR is updated with your proposal. :+1: